relay-event-normalization/src/normalize/span/tag_extraction.rs

```diff
@@ -674,6 +692,23 @@ fn span_op_to_category(op: &str) -> Option<&str> {
     }
 }
 
+/// Reads the event measurements to determine the start type of the event
+fn get_event_start_type(event: Event, is_mobile: bool) -> Option<String> {
+    // Only collect start type for mobile spans
+    if !is_mobile {
+        return None;
+    }
+
+    // Check the measurements on the event to determine what kind of start type the event is
+    if event.measurement("app_start_cold").is_some() {
+        Some("app.start.cold".to_owned())
```

Comment:
nit: Up to you, but the `app.start.` prefix seems redundant to me. Why not just `"cold"` and `"warm"`?

Reply:
Good point, I replaced the values with "cold" and "warm". I see this is also consistent with what's being sent in app context for iOS